btree/btree.c

```diff
@@ -3271,6 +3276,8 @@ static int64_t btree_put_root_split_handle(struct m0_btree_op *bop,
 	/* Increase height by one */
 	tree->t_height++;
 
+	M0_ASSERT(node_expensive_invariant(lev->l_node));
+	M0_ASSERT(node_expensive_invariant(oi->i_extra_node));
```

Comment:
You need to have TBD message here which tells the expensive_invariant call should be removed.

Reply:
done.